packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -30,7 +30,7 @@ export const ChatList = (props: MgtTemplateProps & IChatListItemInteractionProps
     <FluentThemeProvider fluentTheme={FluentTheme}>
       <FluentProvider theme={webLightTheme}>
         {chats.map(c => (
-          <ChatListItem key={c.id} chat={c} myId={chatState.userId} onSelected={props.onSelected} />
+          <ChatListItem key={c.id} chat={{ id: c.id }} myId={chatState.userId} onSelected={props.onSelected} />
```

Comment:
I may be missing this but where is the code to set chatInternal from externally?

Reply:
You don't - you set chat externally. Inside the component, it sets chatInternal to chat and can load additional properties if needed.

Reply:
ChatListItem being a "sub-component" of ChatList would make it safe to share state. However, there is really no reason to share state, using chatInternal seems to work well and is similar to personDetailsInternal which is already used in MGT.